components/file_attachment/file_attachment.tsx

```diff
@@ -104,9 +110,13 @@ export default class FileAttachment extends React.PureComponent {
         }
     }
 
-    onAttachmentClick = (e) => {
+    onAttachmentClick = (e: React.MouseEvent<HTMLAnchorElement|HTMLDivElement, MouseEvent>) => {
         e.preventDefault();
-        e.target.blur();
+
+        if ('blur' in e.target) {
+            (e.target as HTMLAnchorElement).blur();
```

Comment:
Nit: this captures both argument types

```suggestion
            (e.target as HTMLElement).blur();
```

Reply:
Had to adjust the associated test to make it pass with the instanceof check.